client/src/component/MilestoneTable.js

```diff
@@ -1,9 +1,22 @@
-export function MilestoneTable() {
-
+import "../styles/MilestoneTable.css";
+const MilestoneTable = () => {
 	return (
-			<div>
-				<h1>hello world</h1>
-			</div>
+		<div className="table-wrapper">
```

Comment:
i am not sure how we want to name the class we need tp speak about that , for example you use -wraper while I use _container just we need to remind this later to have agreement about for know I think its ok , thank you

Reply:
using underscore( _ ) is not suggested in class naming, cool we can talk about it too

Reply:
Yes we should talk as a team about css. Just remember that wireframes are not designs so you shouldnt be spending time making the design of the MVP match the wireframe